batch_requests/views.py

```diff
@@ -77,10 +78,12 @@ def get_response(wsgi_request):
     if hasattr(response, 'render') and callable(response.render):
         response = response.render()
 
-    if hasattr(response, '_headers'):
+    # Accessing headers method changed in 3.2.0 of Django
+    if django.VERSION < (3, 2, 0):
```

Comment:
just a little fyi, iirc the 0 is superfluous in this check. (ie `django.VERSION < (3, 2)` should be equivalent)